packages/common/src/projects/_internal/ProjectUiSchemaEdit.ts

```diff
@@ -68,6 +68,16 @@ export const uiSchema: IUiSchema = {
             },
           },
         },
+        {
+          labelKey: "{{i18nScope}}.fields.featuredImageAltText.label",
```

Comment:
Remember when I said I added the translation strings somewhere without knowing explicitly how we were going to handle this? Well I put them [here](https://github.com/ArcGIS/opendata-ui/blob/master/packages/hub-components/src/components/arcgis-hub-entity-editor/locales/arcgis-hub-entity-editor.i18n.en.json#L56). So I think you'll want to make this: 
`"{{i18nScope}}.fields.featuredImage.altText.label"`